src/LibraryManager.Vsix/Resources/Text.resx

```diff
@@ -235,4 +235,14 @@ Do you want to continue?</value>
   <data name="Install" xml:space="preserve">
     <value>_Install</value>
   </data>
+  <data name="SuggestedAction_Update_Prerelease" xml:space="preserve">
+    <value>Pre-release: {0}</value>
+  </data>
+  <data name="SuggestedAction_Update_Stable" xml:space="preserve">
+    <value>Stable: {0}</value>
+  </data>
+  <data name="SuggestedAction_Update_Title" xml:space="preserve">
```

Comment:
Should we call it SuggestedAction_Update_Library to keep it consistent with value and other entries above?

Reply:
I wanted to differentiate the use case for this one.  It's the title for the SuggestedActionSet, and isn't actually visible in the UI (though the SuggestedAction API might expose it somewhere).  Since context matters for translations, I didn't want this to seem like a string that could be reused elsewhere.